### PR TITLE
CSS changes in product overview, resized related images, took out border in questions

### PR DESCRIPTION
**Pull request by midorili**

### CODE REVIEW

File: client/src/product/ExpandedView.jsx
Comment (KDD87) [inline client/src/product/ExpandedView.jsx:1]: did you mean to change the import statements back to 'react-icons/Rx'?

File: fec-equinox-ml.pem
Comment (KDD87) [inline fec-equinox-ml.pem:1]: What is this?

**Review by KDD87 — COMMENTED** [review]
Everything looks good! I did have a couple of comments before I merge it.

File: client/src/product/ExpandedView.jsx
Comment (midorili) [reply client/src/product/ExpandedView.jsx:1]: omg, definitely did not mean to do that!! let me fix that and re-commit. good catch!

File: fec-equinox-ml.pem
Comment (midorili) [reply fec-equinox-ml.pem:1]: also yes this is the key required for deployment that should be gitignored.. LOL! let me fix this too

**Review by KDD87 — APPROVED** [review]

**Pull request merged**